### PR TITLE
Illustration: Improve whoops illustration

### PR DESCRIPTION
**Pull request by sirreal**
While reviewing #18397 I noticed that the Whoops svg is imperfect 😱  (see before/after).

This PR:
* Cleans up uneccessary attributes from the svg (`id` and `data-name`)
* Runs the svg through svgo (slightly modified a path to be shorter)
* Replaces a circle-outline-like `path` with a `circle`, which is shorter _and_ more precise.
* Replaces the title which was `itemsFailed` with `Whoops`. It would be nice to get some a11y and i18n feedback about whether `title` has any value here.

## Testing
1. Compare:
   * Before https://wordpress.com/calypso/images/illustrations/whoops.svg
   * After https://calypso.live/calypso/images/illustrations/whoops.svg?branch=update/improve-whoops-svg
1. Better, right?

### Before

before

### After

after

### CODE REVIEW

**Comment by folletto** [comment]
That's by design. 
It's our current illustration style that uses that kind of line.

I agree this specific use is a bit skimpy, and doesn't feel much of an illustration. Maybe @cristelrossignol has some ideas on how to improve this to be a proper illustration?

tl;dr: Please don't merge this! :)

**Comment by sirreal** [comment]
Thanks @folletto. This particular illustration does seem a bit "off" to me, hence the PR.

I'll close so it's clear this shouldn't be merged, but agree it would be nice to have another look at this illustration.

**Comment by sirreal** [comment]
Related: p8hJ04-9G-p2
